src/lib/Libtpp/tpp_internal.h

```diff
@@ -150,20 +150,20 @@ typedef struct {
 typedef struct {
 	pbs_list_head chunks;
 	tpp_chunk_t *curr_chunk;
-	int totlen;
+	size_t totlen;
 	int ref_count;	/* number of accessors */
 } tpp_packet_t;
 
 typedef struct {
-	int ntotlen;
+	unsigned int ntotlen;
```

Comment:
why not use ssize_t here and elsewhere for ntotlen?

Reply:
ssize_t is a signed type, why would you want to use that?

Reply:
That would be changing the data size to long and will render the packets no longer backward compatible. Not required right now.